code/modules/mining/mine_turfs.dm

```diff
@@ -53,7 +53,14 @@ var/list/mining_overlay_cache = list()
 		"carbon" = /obj/item/weapon/ore/coal,
 		"verdantium" = /obj/item/weapon/ore/verdantium,
 		"marble" = /obj/item/weapon/ore/marble,
-		"lead" = /obj/item/weapon/ore/lead
+		"lead" = /obj/item/weapon/ore/lead,
+		"copper" = /obj/item/weapon/ore/copper,
+		"tin" = /obj/item/weapon/ore/tin,
+		"bauxite" = /obj/item/weapon/ore/bauxite,
+//		"void opal" = /obj/item/weapon/ore/void_opal,
+//		"painite" = /obj/item/weapon/ore/painite,
+//		"quartz" = /obj/item/weapon/ore/quartz,
```

Comment:
?

Reply:
I have plans for Void Opal, Painite, and Quartz in future PRs, and I didn't quite want to re-find where I put all the additions the first time. Mining changes are tedious as all hell